src/EnergyPlus/Construction.cc

```diff
@@ -506,10 +505,6 @@ void ConstructionProps::calculateTransferFunction(EnergyPlusData &state, bool &E
                         Nodes(Layer) = ipts1;
                     }
 
-                    if (this->SolutionDimensions > 1) {
-                        if (ipts1 > Construction::MaxCTFTerms / 2) ipts1 = Construction::MaxCTFTerms / 2;
-                    }
-
```

Comment:
[src/EnergyPlus/Construction.cc:510]:(style),[unreadVariable],Variable 'ipts1' is assigned a value that is never used.